lib/Elastica/Query.php

```diff
@@ -23,16 +23,16 @@
 class Query extends Param
 {
     /**
-     * Suggest query or not.
+     * If the current query has a suggest in it.
      *
-     * @var int Suggest
+     * @var bool Suggest
      */
-    protected $_suggest = 0;
+    protected $hasSuggest = false;
 
     /**
      * Creates a query object.
      *
-     * @param array|\Elastica\Query\AbstractQuery $query OPTIONAL Query object (default = null)
+     * @param array|AbstractQuery $query Query object (default = null)
```

Comment:
I had in the past the "Optional" inside to make it more obvious that there is a default when not looking at the code. But it should be obvious enough?

Reply:
Yes, it is obvious as the `$var = null` signature :)